pybtp/btp.py

```diff
@@ -110,6 +110,8 @@
     "passkey_entry_rsp": (defs.BTP_SERVICE_ID_GAP,
                           defs.GAP_PASSKEY_ENTRY,
                           CONTROLLER_INDEX),
+    "start_direct_adv": (defs.BTP_SERVICE_ID_GAP, defs.GAP_START_DIRECT_ADV,
```

Comment:
ditto, BTP spec shall be updated as well

Reply:
There is a Start Directed Advertising command in the BTP spec already.

Reply:
OK :)